Rich-Text-Editor-(RTE)-Overview.md

```diff
@@ -1,117 +1,137 @@
-* Input some plaintext and save. Ensure that the content doesn't disappear upon save.
+
+* Input some plain text and save. Ensure that the content doesn't disappear upon save.
 * Input a single Rich Text Component (such as the Math component) and then save. Make sure it is saved properly.
 * Try using backspace to delete the Rich Text Component. Make sure it is actually deleted.
-* Try copy and pasting a Rich Text Component. Hit save and make sure that the component is copied properly.
-* Try drag and dropping a Rich Text Component from one place in the content to another, then saving. Make sure the component is moved properly.
+* Try copying-and-pasting a Rich Text Component. Hit save and make sure that the component is copied properly.
+* Try editing the Rich Text Component and make sure your changes are shown correctly.
+* Try dragging-and-dropping a Rich Text Component from one place in the content to another, then saving. Make sure the component is moved properly.
 * Input text with multiple paragraphs. Make sure the paragraphs are correctly spaced apart.
 * Insert an Image component with a large image, and save. Make sure the image does not overflow the content boundaries.
 * Insert an Image component with a small image, and save. Make sure the image is not stretched out.
 
-# Rich Text Components
-Oppia explorations can have Rich Text Components, which are custom widgets creators can insert into their content. Each component is an Angular directive, of the form `<oppia-noninteractive-*></oppia-noninteractive-*>`, with different attributes depending on the component. Examples include the Math (for inserting Latex) component, the Image component, and the Video component. This section describes how the components are defined in case you want to add your own component, or need to modify an existing one.
+## Rich Text Components
+
+Oppia explorations can have Rich Text Components, which are custom widgets that creators can insert into their content. Note that we have two kinds of components at play here: rich text components and Angular components. Each rich text component is an Angular component of the form `<oppia-noninteractive-*></oppia-noninteractive-*>`, with different attributes depending on the component. Examples include the Math (for inserting Latex) component, the Image component, and the Video component. This section describes how the components are defined in case you want to add your own component, or need to modify an existing one.
+
+The code for defining the Rich Text Components are housed in [`extensions/rich_text_components`](https://github.com/oppia/oppia/tree/develop/extensions/rich_text_components). In this folder each component has its own folder, with the following files and subfolders:
```

Comment:
are housed --> is housed  (since you're talking about "The code")

Perhaps add comma after "In this folder"

Reply:
done